src/suites/cts/buffers/create_mapped.spec.ts

```diff
@@ -0,0 +1,35 @@
+export const description = ``;
+
+import { TestGroup, pbool, pcombine, poptions } from '../../../framework/index.js';
+
+import { MappingTest } from './mapping_test.js';
+
+export const g = new TestGroup(MappingTest);
+
+g.test('createBufferMapped', async t => {
+  const size = t.params.size;
+  const [buffer, arrayBuffer] = t.device.createBufferMapped({
+    size,
+    usage: GPUBufferUsage.COPY_SRC | (t.params.mappable ? GPUBufferUsage.MAP_WRITE : 0),
+  });
+  await t.checkMapWrite(buffer, arrayBuffer, size);
+}).params(
+  pcombine([
+    poptions('size', [12, 512 * 1024]), //
```

Comment:
nit: Empty comment?

Reply:
It's here to keep the formatter from combining these lines.